static/app/views/dashboardsV2/manage/index.tsx

```diff
@@ -204,18 +252,31 @@ class ManageDashboards extends AsyncView<Props, State> {
               <PageContent>
                 <StyledPageHeader>
                   {t('Dashboards')}
-                  <Button
-                    data-test-id="dashboard-create"
-                    onClick={event => {
-                      event.preventDefault();
-                      this.onCreate();
-                    }}
-                    priority="primary"
-                    icon={<IconAdd size="xs" isCircled />}
-                  >
-                    {t('Create Dashboard')}
-                  </Button>
+                  <ButtonContainer>
+                    <Feature organization={organization} features={['dashboards-edit']}>
+                      <SwitchContainer>
+                        Show Templates
```

Comment:
Oh sorry about one comment reviews, you'll also need to localize the CTAs and titles everywhere
`t('Add Dashboard')`
`t('Show Templates')`
etc

Reply:
```suggestion
                        {t('Show Templates')}
```
Missed a spot!